cmd/neofs-adm/internal/modules/morph/n3client.go

```diff
@@ -47,10 +49,10 @@ type Client interface {
 }
 
 type clientContext struct {
-	Client       Client
-	Hashes       []util.Uint256
-	WaitDuration time.Duration
-	PollInterval time.Duration
+	Client          Client
+	CommitteeAct    *actor.Actor
```

Comment:
We need a short comment here, at least about the `Global` scope.

Reply:
added comments to all the "client" fields